src/content/docs/data-apis/ingest-apis/metric-api/cardinality-management.mdx

```diff
@@ -10,19 +10,19 @@ freshnessValidatedDate: 2024-09-24
   This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
 </Callout>
 
-As an administrator at New Relic, one of your tasks is to make sure you don't go beyond your limit of [GB Ingested](/docs/licenses/license-information/product-definitions/new-relic-one-pricing-definitions/#gb-ingested). One tool for tracking your ingest is the Cardinality Management UI. You'll find helpful graphs and tables explaining your ingest so you can take action if you're approaching limits.
+Cardinality refers to the number of unique elements in a set, specifically the unique attribute combinations observed for a given metric within a one-day period. This data resets daily at UTC midnight. As cardinality increases with more unique combinations, it can impact data limits. Understanding and managing cardinality is crucial for optimizing data usage and preventing limit issues.
 
-## Where can you manage cardinality? [#where-manage]
+As a New Relic administrator, you oversee data ingestion to ensure it stays within the specified limit. <DNT>**Cardinality Management**</DNT> offers graphs and tables to detail your data usage, allowing you to take action if you're approaching your limits.
 
-In the New Relic UI, you can find this with other administration tools. Click your name in the lower left and go to **Administration > Cardinality Management**.
+To view the **<DNT>Cardinality Management</DNT>** UI, go to **<DNT>[one.newrelic.com](https://one.newrelic.com)</DNT> > (click your user name) > <DNT>Administration > Cardinality Management</DNT>**.
 
-## Account Cardinality Breakdown [#account-breakdown]
+## Account cardinality breakdown [#account-breakdown]
 
-If you're in an organization with multiple accounts, you can see a breakdown of cardinality by each account. As with most capabilities within New Relic, you can change the account you are currently viewing by selecting the account picker in the top right corner. The UI will display information about the selected account, as long as you have access to that account. 
+If your organization has multiple accounts, you can view a breakdown of cardinality for each account. To switch accounts, use the account picker in the top right corner. If you have access to the account, you can see the account cardinality information on the UI.
 
 ### Usage over time [#usage-over-time]
 
-On the opening page of **Cardinality Management**, you can view the **Usage over time** chart:
+On the landing page of <DNT>**Cardinality Managemen**t</DNT>, you can view the <DNT>**Usage over time**</DNT> chart. The chart displays the account cardinality usage compared the limit of that account.
 
 <img
```

Comment:
Justify this image

Reply:
This is the first instance in the document where we introduce cardinality management. If a user wants to learn about this capability without logging into New Relic, this screenshot helps them become familiar with it.